lib/index.js

```diff
@@ -80,13 +80,27 @@ let misc_commands = {
       bot.sendMessage(msg.channel, 'Usage: **`!decide`** `something` **`or`** `something...`');
     }
   },
-  drama: (bot, msg) => {
+  drama: (bot, msg, suffix) => {
     let rand = Math.floor(Math.random() * drama.length);
-    bot.sendMessage(msg.channel, drama[rand]);
+    if (suffix >= 0 || suffix <= 25) {
+      bot.sendMessage(msg.channel, drama[suffix]);
+    }
+    if (!suffix || isNaN(suffix) || suffix < 0 || suffix > 25) {
```

Comment:
As you always want to make sure a message is send, I suggest replacing this line with `else` so it'll always end with sending a random drama quote. This way it's also easier to read.

```javascript
if (suffix >= 0 || suffix <= 81) {
  bot.sendMessage(msg.channel, emoji[suffix]);
} else {
  bot.sendMessage(msg.channel, emoji[rand]);
}
```